scripts/deploy

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+build_chart() {
+  docker buildx use default
+  make build
+  
+  docker buildx use m1builder  
```

Comment:
This won't work if you don't have the `m1builder` builder. This is covered by `scripts/package` and the M1 thing is an exception for M1/ARM so if this needs to be included, it needs to be included in the condition that you are on M1/ARM.